src/dodal/devices/i22/NXSasAravis.py

```diff
@@ -0,0 +1,22 @@
+from ophyd_async.core import DirectoryProvider
+from ophyd_async.epics.areadetector import AravisDetector
+
+from dodal.devices.i22.nxsas import NXSasMetadataHolder
+
+
+class NXSasAravis(AravisDetector):
```

Comment:
Can use the existing NXSasOAV class, this implementation doesn't actually read the soft signals.